lucien-ios/LucienConstants.swift

```diff
@@ -9,6 +9,34 @@
 import Foundation
 
 final class LucienConstants {
+
+    // MARK: - Keyboard Constants
+
     static let keyboardHeightPadding = CGFloat(50)
+
+    // MARK: - Form Constants
+
+    static let textFieldBorderOffset = CGFloat(48)
+    static let pickerViewLeftimageEdgeInsetOffset = CGFloat(72)
+
+    // MARK: - Apple Music Blur Effect Constants
```

Comment:
What does Apple Music have to do with these constants? If they just happen to be the same, is that information future developers will need?